broker-daemon/bin/sparkswapd.spec.js

```diff
@@ -77,8 +90,6 @@ describe('sparkswapd', () => {
     expect(BrokerDaemon).to.have.been.calledWith(sinon.match({ rpcAddress }))
   })
 
-  xit('defaults the rpc address to the env variable')
```

Comment:
I tried to retest all of these default env vars again, but was unable to get it to work. I moved the comment up to the top and removed these pending tests